ontologies/CALM_single_MS.json

```diff
@@ -0,0 +1,60 @@
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "id": "e39dg92",
+  "type": "Work",
+  "identifiers": 
+    [
+      {
+        "label": "MS5754",
+        "authority": "CALM"
+      },
+      {
+        "label": "MS.5754",
+        "authority": "CALM"
+      }
+    ],
+  "label": "Oxyrhynchus Papyrus 2547",
+  "creationDate": 
+    {
+      "label": "3rd century AD",
+      "type": "Period"
+    },
+  "description": "<p>Fragment of a document from Oxyrhynchus, Egypt, including a text of the Hippocratic Oath. <p>Recto: A business or official document in a fluent cursive hand, almost entirely illegible. <p>Verso: The Hippocratic Oath. Right hand section of a column of sixteen lines, written in an uncial hand.",
```

Comment:
I know data comes out of CALM with html tags but we don't want that in the ontology, do we?